TestCounterAddLarge fails on some architectures

```
s390x:
=== RUN   TestCounterAddLarge
--- FAIL: TestCounterAddLarge (0.00s)
	counter_test.go:178: valBits expected 18446744073709551616.000000, got 0.000000.
	counter_test.go:181: valInts expected 0, got 18446744073709551615.

ppc64el:
=== RUN   TestCounterAddLarge
--- FAIL: TestCounterAddLarge (0.00s)
	counter_test.go:178: valBits expected 18446744073709551616.000000, got 0.000000.
	counter_test.go:181: valInts expected 0, got 18446744073709551615.

arm64:
=== RUN   TestCounterAddLarge
--- FAIL: TestCounterAddLarge (0.00s)
	counter_test.go:178: valBits expected 18446744073709551616.000000, got 0.000000.
	counter_test.go:181: valInts expected 0, got 18446744073709551615.
```

Otherwise passes on `amd64 i386 armhf`. Not sure if this is indicative of a 64-bit issue on the niche arches.

Version 0.9.0-1

--- a/prometheus/counter_test.go
+++ b/prometheus/counter_test.go
@@ -172,7 +172,7 @@ func TestCounterAddLarge(t *testing.T) {
 	}).(*counter)
 
 	// large overflows the underlying type and should therefore be stored in valBits.
-	large := float64(math.MaxUint64 + 1)
+	large := math.Nextafter(float64(math.MaxUint64), 1e20)
 	counter.Add(large)
 	if expected, got := large, math.Float64frombits(counter.valBits); expected != got {
 		t.Errorf("valBits expected %f, got %f.", expected, got)

